opentelemetry-sdk/src/logs/config.rs

```diff
@@ -2,6 +2,11 @@ use std::borrow::Cow;
 
 use crate::Resource;
 
+/// Default trace configuration
```

Comment:
nit - change the comment above.

Reply:
Also, make this function public as for [trace ](https://github.com/open-telemetry/opentelemetry-rust/blob/1d4db1580fcea89ebd3b3f4e43c5001d8d26cfe7/opentelemetry-sdk/src/trace/mod.rs#L20).